### DISCUSSION TITLE
Need Remote view to inspector game-session in visual (like 3d view BUT only for local)

### DISCUSSION THREAD

**Discussion by jzq740176597** [godotengine/godot-proposals#6597] [Editor]
### Describe the project you are working on

3D

### Describe the problem or limitation you are having in your project

Running Debug / game session I can only **Remote** scene tree and inspector NO visual view to check sth.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

A 3d view to see & check **Remote** stuff can select & drag. rather than just in blind by inspector

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

see above

### If this enhancement will not be used often, can it be worked around with a few lines of script?

no

### Is there a reason why this should be core and not an add-on in the asset library?

core

**Comment by Zireael07** [comment]
Been suggested multiple times and there was even a PR implementing it that was never finished, sadly.

**Comment by theraot** [comment]
Note that you can use the camera icon that is on the top tool bar to override the game camera while it is running in the debugger.

**Comment by Calinou** [comment]
- This is already being tracked in https://github.com/godotengine/godot-proposals/issues/1864.
